assets/js/modules/analytics/components/setup/SetupFormGA4.js

```diff
@@ -16,23 +16,57 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useMount } from 'react-use';
+
 /**
  * WordPress dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { AccountSelect } from '../common';
+import Data from 'googlesitekit-data';
+import { CORE_FORMS } from '../../../../googlesitekit/datastore/forms/constants';
+import { STORE_NAME, PROPERTY_CREATE, FORM_SETUP } from '../../datastore/constants';
+import StoreErrorNotices from '../../../../components/StoreErrorNotices';
+import PropertySelect from '../../../analytics-4/components/common/PropertySelect';
+import AccountSelect from '../common/AccountSelect';
+import GA4PropertyNotice from '../common/GA4PropertyNotice';
+const { useSelect, useDispatch } = Data;
 
 export default function SetupFormGA4() {
+	const accounts = useSelect( ( select ) => select( STORE_NAME ).getAccounts() ) || [];
+
+	const { selectProperty } = useDispatch( STORE_NAME );
+	const { setValues } = useDispatch( CORE_FORMS );
+	useMount( () => {
+		selectProperty( PROPERTY_CREATE );
+		setValues( FORM_SETUP, { profileName: 'All Web Site Data' } );
```

Comment:
Please use the translated default
https://github.com/google/site-kit-wp/blob/524ff2ef0bd77081950e85af5c1b395b72ecc268/assets/js/modules/analytics/components/common/ProfileNameTextField.js#L54